src/frequenz/sdk/timeseries/_quantities.py

```diff
@@ -563,6 +566,9 @@ def __truediv__(self, other: Current) -> Voltage:
 
         Args:
             other: The current to divide by.
+
+        Returns:
+            A voltage from dividing this power by the a current.
```

Comment:
```suggestion
            The voltage from dividing this power by the current.
```

Reply:
These are actually not that well phrased either, it should be `A/The voltage **resulting** from dividing..." or something like that. Maybe we should just use `The calculated voltage` as it is a bit redundant with the method description (which is also wrongly phrased).

I think I will leave these comments for a followup PR improving docs.